Source/WebKit/WebProcess/Cache/WebCacheStorageConnection.cpp

```diff
@@ -64,58 +63,58 @@ void WebCacheStorageConnection::open(const WebCore::ClientOrigin& origin, const
         }
         callback(WTFMove(result));
     };
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::Open(origin, cacheName), WTFMove(newCallback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::OpenCache(origin, cacheName), WTFMove(newCallback));
 }
 
 void WebCacheStorageConnection::remove(WebCore::DOMCacheIdentifier cacheIdentifier, WebCore::DOMCacheEngine::RemoveCacheIdentifierCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::Remove(cacheIdentifier), WTFMove(callback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::RemoveCache(cacheIdentifier), WTFMove(callback));
 }
 
 void WebCacheStorageConnection::retrieveCaches(const WebCore::ClientOrigin& origin, uint64_t updateCounter, WebCore::DOMCacheEngine::CacheInfosCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::Caches(origin, updateCounter), WTFMove(callback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::AllCaches(origin, updateCounter), WTFMove(callback));
 }
 
 void WebCacheStorageConnection::retrieveRecords(WebCore::DOMCacheIdentifier cacheIdentifier, WebCore::RetrieveRecordsOptions&& options, WebCore::DOMCacheEngine::RecordsCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::RetrieveRecords(cacheIdentifier, options), WTFMove(callback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::RetrieveRecords(cacheIdentifier, options), WTFMove(callback));
 }
 
 void WebCacheStorageConnection::batchDeleteOperation(WebCore::DOMCacheIdentifier cacheIdentifier, const WebCore::ResourceRequest& request, WebCore::CacheQueryOptions&& options, WebCore::DOMCacheEngine::RecordIdentifiersCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::DeleteMatchingRecords(cacheIdentifier, request, options), WTFMove(callback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::RemoveRecords(cacheIdentifier, request, options), WTFMove(callback));
 }
 
 void WebCacheStorageConnection::batchPutOperation(WebCore::DOMCacheIdentifier cacheIdentifier, Vector<Record>&& records, WebCore::DOMCacheEngine::RecordIdentifiersCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::CacheStorageEngineConnection::PutRecords(cacheIdentifier, records), WTFMove(callback));
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::PutRecords(cacheIdentifier, records), WTFMove(callback));
 }
 
 void WebCacheStorageConnection::reference(WebCore::DOMCacheIdentifier cacheIdentifier)
 {
     if (!m_connectedIdentifiers.contains(cacheIdentifier))
         return;
 
-    connection().send(Messages::CacheStorageEngineConnection::Reference(cacheIdentifier), 0);
+    connection().send(Messages::NetworkStorageManager::Reference(cacheIdentifier), 0);
```

Comment:
Some of these message names are no longer very meaningful given NetworkStorageManager is not CacheStorageSpecific.
We should probably update the names to make it clear they are CacheStorage related